.github/workflows/build-release.yml

```diff
@@ -50,10 +58,22 @@ jobs:
       run: |
         echo "::set-env name=upload_url::$(cat upload-url/upload-url.txt)"
         echo "::set-env name=asset_file_name::libvips-$(cat LIBVIPS_VERSION)-${{ matrix.platform }}.tar.gz"
+    - name: Install dependencies (macOS)
+      run: |
+        pip3 install meson ninja
+        brew install advancecomp automake nasm pkg-config
+      if: matrix.os == 'macos-latest'
+    - name: Setup toolchain (macOS)
+      shell: bash
+      run: |
+        echo "::set-env name=CC::clang"
+        echo "::set-env name=CXX::clang++"
```

Comment:
I also tried the Clang from LLVM with:
```yml
        echo "::set-env name=CC::/usr/local/opt/llvm/bin/clang"
        echo "::set-env name=CXX::/usr/local/opt/llvm/bin/clang++"
```
and GCC 9 with:
```yml
        echo "::set-env name=CC::gcc-9"
        echo "::set-env name=CXX::g++-9"
```

But this didn't succeed.